code/datums/abilities/wizard.dm

```diff
@@ -221,6 +221,8 @@
 		voice_grim = null
 		voice_fem = null
 		voice_other = null
+		maptext_style = "color: white !important; text-shadow: 1px 1px 3px white; -dm-text-outline: 1px black;"
+		maptext_colors
```

Comment:
please fix and prefix with var/xx while you're here

Reply:
good idea, I went ahead and changed all the vars there to be prefix'd with /var since it's much easier to read than the old way it was done anyways